emulator/memory.cpp

```diff
@@ -54,7 +54,8 @@ pair<uint16_t, uint8_t> memory_control_unit::load_absolute(uint16_t address)
 	else
 	{
 		// Mirror RAM in remaining address space. TODO: write test?
-		return {address, ram[address % RAM_SIZE]};
+		address = address % RAM_SIZE;
+		return {address, ram[address]};
 	}
 }
```

Comment:
Acho que temos que fazer essa parte do mirror ainda:

$2000-$2007 | $0008 | NES PPU registers
$2008-$3FFF | $1FF8 | Mirrors of $2000-2007 (repeats every 8 bytes)

https://wiki.nesdev.com/w/index.php/CPU_memory_map